index.js

```diff
@@ -24,6 +24,8 @@ try {
   const name = core.getInput('name');
   const environment = core.getInput('environment');
   const domain = core.getInput('domain');
+  const output = branchName(github.context).toLowerCase().replace(/[^a-z0-9-]/g, "").substr(0, 63)
+  console.log(`output: ${output}`);
```

Comment:
Keep console.log and output together, just like any other examples in the file...

Reply:
https://github.com/itsmycargo/action-deploy-environment/pull/1/commits/df9e86ab3609d61ae4c077efe92827f3e1d08321